internal/test/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.12
 replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200205145930-e9d93e317dd1 // OpenShift 4.3
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191125132246-f6563a70e19a // OpenShift 4.3
+	github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer => github.com/ravisantoshgudimetla/windows-machine-config-operator/tools/windows-node-installer v0.0.0-20200221233526-91c18959244f
```

Comment:
I think this useful for everyone to know. This is how we can prove that it works in our CI in case our PR related to other go mod did not merge